src/assets/stylesheets/base/_buttons.scss

```diff
@@ -0,0 +1,8 @@
+button {
+  border: transparent;
+
+  background-color: map-get($brand-white, 10);
+  
+  color: #15151a;
```

Comment:
we should get this color from a variable.

Reply:
Added in 8d166ade58218d847fdaccc7c0f0aaba0389d7a4